app/javascript/components/FiltersBox.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect } from 'react';
+import Button from 'react-bootstrap/Button';
+import InputGroup from 'react-bootstrap/InputGroup';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+
+import FiltersSearchBar from './FiltersSearchBar';
+
```

Comment:
I would use lodash here instead of rolling our own array compare (it looks like it's already in our yarn.lock file) .  Lodash also supports partials imports so if it's not already in our package, we only have to bring in the functions we need


>  __.isEqual(array1, array2)

Reply:
Good point, done in 69208ca.